horovod/spark/lightning/estimator.py

```diff
@@ -170,7 +170,13 @@ class TorchEstimator(HorovodEstimator, TorchEstimatorParamsWritable,
     logger = Param(Params._dummy(), 'logger', 'optional logger')
 
     log_every_n_steps = Param(Params._dummy(), 'log_every_n_steps', 'control the frequency of logging',
-                                typeConverter=TypeConverters.toInt)
+                              typeConverter=TypeConverters.toInt)
+
+    data_loader_class = Param(Params._dummy(), 'data_loader_class',
```

Comment:
Is this a name (string) or is it a class / function? Seems it would be easier to pass around a class directly.